data/slide_deck/mutations/create_slide_deck.py

```diff
@@ -138,21 +138,21 @@ def mutate(self,
 
         # problem statement
 
-        slide = presentation.slides.add_slide(plain_layout)
-        title_shape = CreateSlideDeck.get_placeholder(slide, 'Title 2')
-        body_shape = CreateSlideDeck.get_placeholder(slide, 'Text Placeholder 1')
-        title_shape.text = "Problem Statement"
-        body_shape.text = problem_statement
-        CreateSlideDeck.add_notes(slide)
+        # slide = presentation.slides.add_slide(plain_layout)
```

Comment:
@hafen Can these lines be deleted or do we expect them to come back soon?

Reply:
I have removed them.